### ISSUE TITLE
State transition table not implemented

### ISSUE THREAD

**Issue by SidhraFernando-Plant** [automatarium/automatarium#280]
Transition table is not implemented for any type of automata. The states and alphabet appear as added, but the cell values never change. **Either remove the table or implement the table** depending on how difficult implementation will be and the priority of having the table functional. Imo it would be nice to have but lower priority than getting the other core features working for a deployable version. Screenshots of empty tables for all automata (see right panel):

<img width="1629" alt="Screen Shot 2023-03-12 at 9 43 10 pm" src="https://user-images.githubusercontent.com/79517325/224539629-d88a695d-28f2-44bb-b4eb-4a29fce51e39.png">
<img width="1631" alt="Screen Shot 2023-03-12 at 9 43 24 pm" src="https://user-images.githubusercontent.com/79517325/224539632-64daea99-125f-4a4d-9c14-1cd3567dc6d0.png">
<img width="1631" alt="Screen Shot 2023-03-12 at 9 43 44 pm" src="https://user-images.githubusercontent.com/79517325/224539633-9827936f-1a33-4ed1-b057-d18ac7390088.png">

**Comment by SidhraFernando-Plant** [comment]
Oh that's great @tdib im sure it will help a lot 😁

**Comment by ire4ever1190** [comment]
Looks like the issue was caused by me.

The part that builds the table relies on implicit casting to compare (which got changed to `===` to please the linter)
Making this line
https://github.com/automatarium/automatarium/blob/56feb729b3b8a33f6d48101cc0480f8849322dc9/frontend/src/components/Sidepanel/Panels/Info/Info.js#L77
into this
```js
        .filter(([key]) => key.split(',')[0] === state.id.toString() && key.split(',')[1] === symbol)
```
Should fix it (Or reverting back to using `==`, but imo properly converting seems better)

**Comment by ire4ever1190** [comment]
Closed by #304
